frontend/src/scenes/funnels/FunnelLineGraph.tsx

```diff
@@ -29,6 +29,7 @@ export function FunnelLineGraph({
             insightId={insight.id}
             inSharedMode={!!inSharedMode}
             percentage={true}
+            aggregationGroupTypeIndex={filters.aggregation_group_type_index}
```

Comment:
Types don't line up here - this can be number | null, `number` on the other end?

Reply:
Hm it looks like both are typed to `number | undefined` so shouldn't the types here be the same?